### PR TITLE
A proposal for a security access resource

### PR DESCRIPTION
**Pull request by michael-mclawhorn**

### CODE REVIEW

File: rfc/permissions.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+[USERNAME] - refers to the authenticated name on the security context principal.  
+This could be provided by an external auth layer based on the application
+authentication mechanism. 
+
+/users/[USERNAME]/access  - Describes the resources in the application 
+which the user can query.
+
+Sample document:
+{
+   tables: { "tablename1", "tablename2" }
```
[the remark this answers is not in the document]
Comment (archolewa) [reply rfc/permissions.md]: Why `tableGroup`? That's an internal implementation detail. Users shouldn't have to care about that.

File: rfc/permissions.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    username: [USERNAME],
+    access: {
+        tables: { "tablename1", "tablename2" }
+    }
+}
+
+
+Another possible extension:
+
+/user -> alias to /user/[USERNAME FOR AUTHENTICTED_USER]
```
Comment (michael-mclawhorn) [inline rfc/permissions.md]: correct.  Could use /self as the alias.

File: rfc/permissions.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+This could be used as a white list or black list.  The presumption would
+probably be that an unmentioned resource is fully available.
+
+/users/[USERNAME]
+
+Sample document:
+{
+    username: [USERNAME],
+    access: {
+        tables: { "tablename1", "tablename2" }
```
[the remark this answers is not in the document]
Comment (michael-mclawhorn) [reply rfc/permissions.md]: List of objects not strings.

File: rfc/permissions.md (hunk 2 above)
Comment (michael-mclawhorn) [reply rfc/permissions.md]: NEEDS GOOD DOCS, be aware of type system in REST (for Json API clients)

**Comment by michael-mclawhorn** [comment]
This change has failed to garner a constituency

**Pull request closed without merging**